src/db/mysql/db.sql

```diff
@@ -50,6 +50,7 @@ CREATE TABLE volume (
    compress_blobs         BOOLEAN NOT NULL,
    compression_threshold  BIGINT NOT NULL,
    metadata               MEDIUMTEXT,
+   store_type             TINYINT(1) NOT NULL DEFAULT 1, -- 1 for onstore and 2 for s3 bucket
```

Comment:
Change comment text to '1 for INTERNAL(onstore) and 2 for EXTERNAL(s3 bucket, OPENIO)

Reply:
Done